src/components/textbox/textbox.component.tsx

```diff
@@ -79,7 +79,10 @@ export interface CommonTextboxProps
   label?: string;
   /** Inline label alignment */
   labelAlign?: "left" | "right";
-  /** [Legacy] A message that the Help component will display */
+  /** [Legacy] Text applied to label help tooltip. When opted into new design validations
```

Comment:
```
/** [Legacy] Text applied to label help tooltip. When opted into new design validations
   * it will render as a hint above the input, unless an `inputHint`
   * prop is also passed
   */